controller/argument_value_resolver.rst

```diff
@@ -51,12 +55,12 @@ controller::
 
     namespace AppBundle\Controller;
 
-    use AppBundle\Entity\User;
     use Symfony\Component\HttpFoundation\Response;
+    use Symfony\Component\Security\Core\User\UserInterface;
 
     class UserController
     {
-        public function indexAction(User $user)
+        public function indexAction(UserInterface $user)
```

Comment:
Good point. I'll update the example to make it consistent. Both the controller argument and the ArgumentValueResolver should use the UserInterface.